java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -170,6 +175,22 @@ public ColumnVector(DType type, long rows, Optional<Long> nullCount,
       assert offsetBuffer == null : "offsets are only supported for STRING";
     }
 
+    long[] children = new long[] {};
```

Comment:
This constructor should be implemented in terms of the new `DataType` constructor version.